frappe/public/js/legacy/form.js

```diff
@@ -919,12 +919,23 @@ _f.Frm.prototype.add_custom_button = function(label, fn, group) {
 	return btn;
 };
 
+//Remove all custom buttons
 _f.Frm.prototype.clear_custom_buttons = function() {
 	this.page.clear_inner_toolbar();
 	this.page.clear_user_actions();
 	this.custom_buttons = {};
 };
 
+//Remove specific custom button by button Label
+_f.Frm.prototype.remove_custom_buttons = function(buttons) {
```

Comment:
can you add an example how this is used?

Also better if the method is `remove_custom_button` (single) and the argument can be a single button or list.

Reply:
@rmehta updated with example